angel.py

```diff
@@ -210,21 +212,25 @@ def difference_operator_lists(li1, li2):  # Used to find out the rejected player
     print(f"tAngel 2021 engine initializing..............")
     print(f"=============================================")
     print(f"\n\n")
+    # Read config_file to obtain the header column header names
+    config_file = open(CONFIGFILE, 'r')
+    column_names = yaml.full_load(config_file)["player_attribute_index"]
```

Comment:
If we would use this file later for other purposes later, you might store it before indexing ["player_attribute_index"]

Reply:
What would happen if ["player_attribute_index"] were null? Or an array? Or a string?

Reply:
i added some error management